geoips/interface_modules/algorithms/pmw_tb/pmw_color37.py

```diff
@@ -18,7 +18,9 @@
 
 LOG = logging.getLogger(__name__)
 
+family = "list_numpy_to_numpy"
```

Comment:
@jsolbrig Does this work with JUST alg_func_type for now?  So it will work with just family, just alg_func_type, or both family and alg_func_type at the moment?

Reply:
At the moment, this will look for `family` and if it isn't found will fall back to `alg_func_type`.